lib/uklock/include/uk/mutex.h

```diff
@@ -43,6 +43,9 @@
 #include <uk/wait_types.h>
 #include <uk/plat/time.h>
 
+#include <uk/arch/spinlock.h>
+#include <string.h>
```

Comment:
Those might be worth protecting with `#ifdef CONFIG_LIBUKLOCK_MUTEX_METRICS` guards